### PR TITLE
Integrate header

### PR DESCRIPTION
**Pull request by lakshmishriaswathnarayana**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #KAW-7679

Test URLs:
- Before: https://main--edge-delivery-bimota--netcentric.hlx.page/
- After: https://KAW-7679-header--edge-delivery-bimota--netcentric.hlx.page/

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -131,145 +230,196 @@ header nav[aria-expanded='true'] .nav-hamburger-icon::after {
...
+/* nav-drop */
+header .nav-drop {
+  position: relative;
+  cursor: pointer;
 }
 
-header nav .nav-brand img {
-  width: 128px;
-  height: auto;
+header .nav-drop::after {
```
Comment (TomaszDziezykNetcentric) [inline blocks/header/header.css:241]: The icon is not aligned properly on mobile
obraz

File: blocks/header/header.js [hunk 2]
```diff
@@ -116,34 +140,79 @@ export default async function decorate(block) {
...
 
   const navWrapper = document.createElement('div');
   navWrapper.className = 'nav-wrapper';
-  navWrapper.append(nav);
+  navWrapper.append(mobileLogoWrapper, nav);
   block.append(navWrapper);
+
+  checkForActiveLink(navSections);
+
+  const observer = new IntersectionObserver((entries) => {
```
Comment (TomaszDziezykNetcentric) [inline blocks/header/header.js:208]: It's not showing the top image (2 red and 2 black strokes) when the initial resolution is desktop.

File: blocks/header/header.css (hunk 1 above)
Comment (TomaszDziezykNetcentric) [reply blocks/header/header.css:241]: And it is overlapping for 375px devices

**Review by TomaszDziezykNetcentric — CHANGES_REQUESTED** [review]

File: blocks/header/header.js (hunk 2 above)
Comment (lakshmishriaswathnarayana) [reply blocks/header/header.js:208]: This expected behaviour

**Review by TomaszDziezykNetcentric — APPROVED** [review]

**Pull request merged**